src/view/com/posts/PostFeedItem.tsx

```diff
@@ -1,31 +1,31 @@
 import React, {memo, useMemo, useState} from 'react'
 import {StyleSheet, View} from 'react-native'
 import {
-  AppBskyActorDefs,
+  type AppBskyActorDefs,
```

Comment:
d'oh- rebase broke this?

Reply:
.. never mind! that's just a side effect of f6f253b4c93f5166648615d03f38ede40135f646